eventsextras.php

```diff
@@ -2,6 +2,27 @@
 
 require_once 'eventsextras.civix.php';
 use CRM_EventsExtras_ExtensionUtil as E;
+use CRM_EventsExtras_SettingsManager as SettingsManager;
+
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @link hhttps://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_buildForm/
+ */
+function eventsextras_civicrm_buildForm($formName, &$form){
+  if ($formName === 'CRM_Event_Form_ManageEvent_EventInfo'){
+    $eventInfoHook = new CRM_EventsExtras_Hook_BuildForm_EventInfo(SettingsManager::EVENT_INFO);
```

Comment:
You don't need to pass this in from the constructor. Each extending class can have a private property that defines what the value of $eventTab is.